File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// BluefinDecryptxP2pe SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class BluefinDecryptxP2peMakeContext
{
    public static function call(array $ctxmap, ?BluefinDecryptxP2peContext $basectx): BluefinDecryptxP2peContext
    {
        return new BluefinDecryptxP2peContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/rb/utility/make_context.rb
# BluefinDecryptxP2pe SDK utility: make_context
require_relative '../core/context'
module BluefinDecryptxP2peUtilities
  MakeContext = ->(ctxmap, basectx) {
    BluefinDecryptxP2peContext.new(ctxmap, basectx)
  }
end
